src/analytics/docs.ts

```diff
@@ -316,10 +316,6 @@ export const eventDocs: Record<AnalyticsEventType, string> = {
   [FiatExchangeEvents.cico_landing_select_flow]: `User taps which CICO flow they want (Cash in, cash out, spend)`,
   [FiatExchangeEvents.cico_landing_how_to_fund]: `User taps help article link at the bottom`,
 
-  // The CICO Currency screen where the user selects their digital currency
-  [FiatExchangeEvents.cico_currency_chosen]: `User selects a currency and taps next`,
-  [FiatExchangeEvents.cico_currency_back]: `User presses the back button from the currency screen`,
```

Comment:
there is a section at the bottom that is commented out with deleted events. kinda helps for when someone asks where an event comes from since we can tell it's from a past version of the app. can you add these to that section?

Reply:
Added in 99d0df6989e080ace8b547907abf82da6413d5e4!